gatsby/content/blog/2021/12/2021-12-03-sydent-typing-1.mdx

```diff
@@ -0,0 +1,187 @@
+---
+  date: '2021-12-03'
+  title: 'Type coverage for Sydent: motivation'
+  categories:
+    - Thoughts
+    - Tech
```

Comment:
Probably _just_ Tech. The Thoughts category is mostly Matthew musing about broader things, so not entirely sure this fits the theme

Reply:
This is 3b9150e5.